dates_test.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime, timedelta
+
+import pytest
+
+import dates
+
```

Comment:
That's a lot of whitespace
```suggestion
import dates
```